### PR TITLE
Update(Examples): Discord.js v13

### PR DESCRIPTION
**Pull request by Grayson-code**
Updated the code to run with latest versions

### CODE REVIEW

**Review by Anish-Shobith — APPROVED** [review]
Looks good to me to reflect discord.js v13 changes

**Comment by Grayson-code** [comment]
> Looks good to me to reflect discord.js v13 changes

Thanks!

**Comment by Anish-Shobith** [comment]
@Grayson-code can you make changes in different files too, I am sure they are more basically for example `index.js` which contains requirements of passing intents too.

**Comment by Grayson-code** [comment]
> @Grayson-code can you make changes in different files too, I am sure they are more basically for example `index.js` which contains requirements of passing intents too.
> 
> Because they are also required and equally important,you can go about changing the pull request name to reflect v13 changes.

yeeee am looking into more files now only so yeajh

**Comment by Grayson-code** [comment]
i believe I have made changes to example/* to work with discord.js v13

**Comment by Anish-Shobith** [comment]
Could you follow the code style, more information here : https://app.codacy.com/gh/MenuDocs/erela.js/pullRequest?prid=8793096 
If you feel if it's a mistake at some situations please disable eslint rules

File: examples/index.js [hunk 1]
```diff
@@ -2,37 +2,46 @@ const { Client, Collection } = require("discord.js");
 const { readdirSync } = require("fs");
 const { Manager } = require("erela.js");
 
-const client = new Client();
+const client = new Client({
+  intents: 32703,
```
Comment (Anish-Shobith) [inline examples/index.js:6]: Can you explain what's intent `32703` is with a comment

Comment (Grayson-code) [reply examples/index.js:6]:
```
GUILDS 
GUILD_MEMBERS
GUILD_BANS
GUILD_EMOJIS_AND_STICKERS 
GUILD_INTEGRATIONS 
GUILD_WEBHOOKS 
GUILD_VOICE_STATES
GUILD_PRESENCES 
GUILD_MESSAGES 
GUILD_MESSAGE_REACTIONS 
GUILD_MESSAGE_TYPING 
DIRECT_MESSAGES 
DIRECT_MESSAGE_REACTIONS 
DIRECT_MESSAGE_TYPING 

```
 i am going to update the intents and remove the unused intents

**Comment by Grayson-code** [comment]
> Could you follow the code style, more information here : https://app.codacy.com/gh/MenuDocs/erela.js/pullRequest?prid=8793096 If you feel if it's a mistake at some situations please disable eslint rules

sure gotcha

**Comment by Grayson-code** [comment]
i worked with the code style but has left the issues that were there before PR

**Comment by Grayson-code** [comment]
Also about the https://solaris.codes domain in the repo as its expired would you like to replace all of em with like the current netlify link?

**Comment by melike2d** [comment]
superseded by #184

**Pull request closed without merging**